sentry_streams/sentry_streams/examples/events.py

```diff
@@ -31,37 +56,96 @@ def build_event(value: str) -> Event:
 
     d: dict[str, Any] = json.loads(value)
 
-    return Event(d["project_id"], d["latency"], d["timestamp"])
+    return Event(d["project_id"], d["latency"], d["timestamp"], d["tags"], d["value"])
 
 
-def build_alert_json(alert: AlertData) -> str:
+def build_alert_json(alert: Union[p95AlertData, CountAlertData]) -> str:
 
     d = alert.to_dict()
 
     return json.dumps(d)
 
 
-class AlertsBuffer(Accumulator[Event, AlertData]):
+class AlertsBuffer(Accumulator[AlertingEvent, Union[p95AlertData, CountAlertData]]):
```

Comment:
We are going to have an instance of AlertsBuffer per group by key value, right? Which means, per alert id.